Formula/rustfmt.rb

```diff
@@ -0,0 +1,29 @@
+class Rustfmt < Formula
+  desc "Format Rust code"
+  homepage "https://rust-lang.github.io/rustfmt/"
+  url "https://github.com/rust-lang/rustfmt/archive/refs/tags/v1.4.38.tar.gz"
+  sha256 "4856cac540a6d894bc019285b690cebd2076aa8067bd8509ec176645c47a31bf"
+  license any_of: ["MIT", "Apache-2.0"]
+  head "https://github.com/rust-lang/rustfmt.git", branch: "master"
+
+  depends_on "rustup-init" => :build
+  depends_on "rust" => :test
+
+  def install
+    system "#{Formula["rustup-init"].bin}/rustup-init", "-qy", "--no-modify-path"
+    ENV.prepend_path "PATH", HOMEBREW_CACHE/"cargo_cache/bin"
+    # we are using nightly because rustfmt requires nightly in order to build from source
+    # pinning to nightly-2021-11-08 to avoid inconstency
+    system "rustup", "toolchain", "install", "nightly-2021-11-08"
+    system "rustup", "component", "add", "rust-src", "rustc-dev", "llvm-tools-preview", \
+      "--toolchain", "nightly-2021-11-08"
```

Comment:
```suggestion
    system "rustup", "component", "add", "rust-src", "rustc-dev", "llvm-tools-preview",
           "--toolchain", "nightly-2021-11-08"
```
Let's align the arguments to `system`, and we don't need the `\`.